services/web/server/src/simcore_service_webserver/trash/_service.py

```diff
@@ -115,6 +153,9 @@ async def safe_empty_trash(
     # Delete explicitly trashed folders (and all implicitly trashed sub-folders and projects)
     await _empty_explicitly_trashed_folders_and_content(app, product_name, user_id)
 
+    # Delete explicitly trashed workspaces (and all implicitly trashed sub-folders and projects)
+    await _empty_explicitely_trashed_workspaces_and_content(app, product_name, user_id)
```

Comment:
is this equivalent to `Empty Trash (delete permanently)`? it looks less scarry than the previous one ;)